lib/application.rb

```diff
@@ -13,6 +13,7 @@ class Application < Sinatra::Base
   end
 
   get '/shopping_list' do
-    ListItem.all.map(&:name).join("\n")
+    @list_items = ListItem.all.map(&:name)
```

Comment:
maybe just `ListItem.all`, and then `item.name` at https://github.com/romper-collective/witchcult.shop/pull/10/files#diff-b86d3fd25408b9c06a031a57f52b4ae9R7 ?